src/modules/ModuleRunner.ts

```diff
@@ -29,6 +33,13 @@ import "./ModuleComponents";
 export class ModuleRunner {
     public static readonly instance = new ModuleRunner();
 
+    public className: string = ModuleRunner.name;
+
+    public extensions: AllExtensions = {
+        cryptoSetup: new DefaultCryptoSetupExtensions(),
+        experimental: new DefaultExperimentalExtensions(),
+    };
```

Comment:
It seems a mistake to make this a public property that anywhere in the codebase can write to. Better to make it a `private` property, and provide read-only accessors to `cryptoSetup` and `experimental`.